nearcore/src/runtime/mod.rs

```diff
@@ -1646,6 +1651,14 @@ impl RuntimeAdapter for NightshadeRuntime {
             epoch_manager.get_prev_epoch_id(prev_block_hash).map_err(Error::from)
         }
     }
+
+    fn will_shard_layout_change(&self, parent_hash: &CryptoHash) -> Result<bool, Error> {
+        let mut epoch_manager = self.epoch_manager.as_ref().write().expect(POISONED_LOCK_ERR);
+        Ok(epoch_manager
+            .get_split_shards_if_shards_will_change(parent_hash, vec![])
+            .map_err(Error::from)?
```

Comment:
This should be enough since `?` operator uses Into trait anyway:

```suggestion
            .get_split_shards_if_shards_will_change(parent_hash, vec![])?
```